src/ex07.cairo

```diff
@@ -5,8 +5,13 @@
 // - Use this contract's claim_points() function
 // - Your points are credited by the contract
 ////////////////////////////////
+#[starknet::interface] 
+trait Ex07Trait<T> {
```

Comment:
The first argument to your Trait functions should be a reference to the Contracts state as seen here. Also, review the trait implementation.

https://book.starknet.io/chapter_2/functions.html#traits

Reply:
Thanks, I've adjusted the code accordingly